tests/app/helpers.py

```diff
@@ -280,6 +280,15 @@ def set_framework_status(self, slug, status):
         Framework.query.filter_by(slug=slug).update({'status': status})
         db.session.commit()
 
+    def set_framework_variation(self, slug):
+        Framework.query.filter_by(slug=slug).update({
+            'framework_agreement_details': {
+                "frameworkAgreementVersion": "v1.0",
+                "variations": {"1": {"createdAt": "2016-08-19T15:31:00.000000Z"}}
+            }
+        })
+        db.session.commit()
+
```

Comment:
Now here's where the two approaches to test fixtures start overlapping with each other. (@pcraig3  &) I created a bunch of pytest fixtures that amongst other things creates frameworks with variations (see `conftest.py`)